File: Daily/Rust/Leetcode374-猜数字大小.rs
impl Solution {
    unsafe fn guessNumber(n: i32) -> i32 {
        let (mut l,mut r) = (1,n);
        while l < r{
            let mid = l + (r - l) / 2;
            if(guess(mid) > 0){
                l = mid + 1;
            }else{
                r = mid;
            }
        }  
        l
    }
}

File: Leetcode/PY/Leetcode374-猜数字大小.py
class Solution:
    def guessNumber(self, n: int) -> int:
        l,r = 1,n
        while l < r:
            mid = (l + r) >> 1
            if guess(mid) > 0:
                l = mid + 1
            else:
                r = mid 
        return l